projects/lib/src/socialauth.service.ts

```diff
@@ -98,6 +101,34 @@ export class SocialAuthService {
       });
   }
 
+  refreshAuthToken(providerId: string): Promise<SocialUser> {
```

Comment:
I think we can have the `getLoginStatus` method exposed through this, rather than just `refreshAuthToken`.

Reply:
Good point. Now  `refreshAuthToken` is returning `void` as a promise and broadcast next with new `_authState`.